src/components/shared/Datatable.vue

```diff
@@ -34,11 +34,11 @@
                     scopedSlots: this.$scopedSlots,
                     ref: 'vuetable',
                     on: {
-                        ...this.$listeners,
                         'vuetable:pagination-data': this.onPaginationData,
                         'vuetable:cell-clicked': this.onCellClicked,
                         'vuetable:loading': () => { this.loading = true },
                         'vuetable:loaded': () => { this.loading = false },
+                        ...this.$listeners,
```

Comment:
This change allows me to overide the `vuetable:cell-clicked` default method which opens the detail row... in the case of task creds, i only want to show the detail row if a session exists so need more logic in the event listener method... If you prefer i can tweak this change to only allow overiding this single listener rather than all of them.. let me know

Reply:
No this makes sense. These are only here for convenience so no reason why they can't be overriden, as long as the override doesn't break the base functionality.